web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts

```diff
@@ -61,11 +60,16 @@ const settings = {
   },
   outputPermissions: {
     label: 'Output Permissions',
-    field: 'output_permissions',
-    type: 'comma-list',
+    field: 'outputPermissions',
     description: 'A comma separated list of (role,capability) pairs to apply to loaded documents.\nDefault: The default permissions associated with the user inserting the document.\n\nExample: -output_permissions role1,read,role2,update',
     value: 'rest-reader,read,rest-writer,update',
   },
+  outputURIReplacement: {
+    label: 'Output URI Replacement',
+    field: 'outputURIReplacement',
+    description: 'Specify a prefix to prepend to the default URI. Used to construct output document URIs. For details, see Controlling Database URIs During Ingestion.',
+    value: ''
+  }
```

Comment:
Also this property essentially performs one or more string substitutions on the default URI. The description has to be changed to reflect that

Reply:
@srinathgit I took the description from previous mlcp based ingestion 1-to-1, it consistent with your description from Friday, everything else should be in the docs: 'Controlling Database URIs During Ingestion'. Also, I am not sure I understand the above description.

Reply:
https://docs.marklogic.com/guide/mlcp/import#id_42798. The property's functionality corresponds to "output_uri_replace" in mlcp.